main.py

```diff
@@ -10,36 +10,30 @@
 port = '8989' #YOUR SONARR PORT
 #END
 
-ignore_default_episdode_name = True # Ignore files where the Episode uses default episode naming?
-scrub__and__strings = True # scrub 'and' from file name strings. Fixes & vs 'and' mismatches
+IGNORE_DEFAULT_EPISODE_NAME = True # Ignore files where the Episode uses default episode naming?
+SCRUB__AND__STRINGS = True # scrub 'and' from file name strings. Fixes & vs 'and' mismatches
 
 # If a file covers multiple episodes, but the episodes otherwise have the same (eg Part1 & Part2), this will allow these to be handled better
 # Multi episode files where the episode name is different should be handled OK now anyway, (once renamed to Sonarr standards), as both episode names
 # should exist in the file name 
 # Also, episode must be in the format SnnExx-yy)
-allow_multi_part_episode_files = True
+ALLOW_MULIT_PART_EPISODE_FILES = True
 
 
 z=0 # global counter
 
 
 def default_episode(title):
     """Check whether the episode title has generic naming applied."""
-    if title[:7] == "Episode" and str.isnumeric(title[8:]):
-        return True
-    
-    return False
+    return (title[:7] == "Episode" and str.isnumeric(title[8:]))
```

Comment:
The if statement already does the work of the two return statements so why not just return the conditional. (this was a big pet peeve of one of my CS professors)

Reply:
makes sense, thanks!